src/libraries/System.Private.Xml/src/System/Xml/Core/XmlTextReaderImpl.cs

```diff
@@ -4200,7 +4196,6 @@ private bool ParseDocumentContent()
                 // read new characters into the buffer
                 if (ReadData() != 0)
                 {
-                    pos = _ps.charPos;
```

Comment:
The if condition can now be inverted and the if block removed.

Reply:
Done